src/main/resources/templates/layout.html

```diff
@@ -36,7 +37,7 @@
                     <div class="input-group mr-sm-2">
                         <input class="form-control" type="text" placeholder="Search" aria-label="Search">
                         <div class="input-group-append">
-                            <button class="btn btn-secondary" type="submit"><i class="fas fa-search"></i></button>
+                            <button class="btn btn-secondary" type="submit"><i class="fas fa-search" ></i></button>
```

Comment:
Seems like accidentally added space.
```suggestion
                            <button class="btn btn-secondary" type="submit"><i class="fas fa-search"></i></button>
```